app/src/main/resources/script.js

```diff
@@ -1,38 +1,26 @@
-function shippingCb(address, tokenCallback) {
-    tokenCallback({ // Can return price based on address
-        shippingMethods: [
-            {
-                id: '0',
-                name: 'Standard Ground (5-9 business days)',
-                deliveryTime: '5 - 9 Business Days',
-                cost: 0
-            },
-        ],
-        tax: 0
-    });
+function initiatePayment() {
+    var XHR = new XMLHttpRequest();
+
+    // Set up our request
+    XHR.open('POST', 'http://localhost:3000/transfer', true);
+
+    XHR.setRequestHeader("Content-Type", "application/json; charset=utf-8");
+
+    var data = $.param({
```

Comment:
if you are relying on jQuery for this, why not use $.post instead of native XMLHttpRequest?

Reply:
There were problems with the redirect part when I was using $.post, ended up with preflight response errors from the web app. This way didn't cause those errors, could be worth playing with to make it consistent though.